### DISCUSSION TITLE
My screen automatically switch to 60Hz when I turn on my computer

### DISCUSSION THREAD

**Discussion by Haolambor** [seerge/g-helper#3242] [Q&A] [answered]
Why does my screen automatically switch to 60Hz when I turn on my computer, even though I set it to 144Hz with Overdrive?

**Comment by Haolambor** [comment]
![Screenshot 2024-10-09 232044](https://github.com/user-attachments/assets/18bf7116-75cb-4df4-88a3-89c85db6e507)

**Comment by seerge** [comment, marked as answer]
@Haolambor hello, I don't know :) But it's not G-Helper doing that unless you select `Auto` mode or manually click on `60Hz` button.

If you have [Asus Smart Display Control](https://github.com/seerge/g-helper/wiki/Requirements#recommendations-optional) installed, it may be interfering so you need to uninstall it.

**Reply by Haolambor** [reply]
Thank you very much, the above solution has solved my problem.

**Reply by Haolambor** [reply]
By the way, what is the 'overdrive' feature for?

**Reply by seerge** [reply]
It reduces display response time and enabled on most of asus laptops by default
